src/index.ts

```diff
@@ -15,6 +15,8 @@ export * from './morphology';
 export * from './operations';
 export * from './roi';
 export * from './save';
+export * from './Stack';
+export * from './stack/index';
```

Comment:
For now, do not export all these methods and only export the `Stack` class.

It is confusing to have all those stack-specific function with generic names export at the first level.

Eventually we would want to import stack-related functions in a namespace: `import {xxx} from 'image-js/stack'`